playout/setup.py

```diff
@@ -43,5 +43,11 @@
         "pytz",
         "requests",
     ],
+    extras_require={
+        "dev": [
+            f"libretime-shared @ file://localhost/{here.parent / 'shared'}#egg=libretime_shared",
+            f"libretime-api-client @ file://localhost/{here.parent / 'api_client'}#egg=libretime_api_client",
+        ],
+    },
```

Comment:
I would rather have this in the main install_requires, but it seems this leads to problems when installing for production.

I think its a pip version issue, but maybe we could publish these package to pip ?

Reply:
We can definitely publish to pip. The only issue there is that it won't help with development versions. pip packages will need to be versioned, so can publish as part of the release CI, but development versions will still need to install from file

Reply:
This need a proper discussion, as I would even go for publishing each app to pypi. I'll open a ticket.